src/inmanta/protocol/openapi/converter.py

```diff
@@ -49,10 +49,10 @@
 from inmanta.types import ReturnTypes
 
 
-def openapi_json_encoder(o: object) -> Union[ReturnTypes, util.JSONSerializable]:
+def openapi_json_encoder(o: object, tz_aware: bool = True) -> Union[ReturnTypes, util.JSONSerializable]:
```

Comment:
I don't really see a need for the parameter here if we never pass it explicitly. I would revert the changes to this file.